portfolio/src/main/webapp/script.js

```diff
@@ -63,14 +63,21 @@ function previousExperience() {
   selectedExperience--;
 }
 
-function getServerComments() {
-  fetch('/data').then(response => response.json()).then((commentSection) => {
+function getNumCommentsSelected() {
+  var comments = document.getElementById('number-comments');
+  var numCommentsSelected = comments.options[comments.selectedIndex].value;
+  getServerComments(numCommentsSelected);
+}
```

Comment:
This ties into Kristen's comment from earlier but you could save yourself a bit of work here by having this method accept an index parameter and then in your HTML, doing this:

```
<select onchange="getNumCommentsSelected(this.selectedIndex)" (....)>
```

Reply:
Thanks for all the feedback, this all helped a lot!